src/user-guide/installing-workflows.rst

```diff
@@ -0,0 +1,370 @@
+.. _Installing-workflows:
+
+Installing Workflows
+====================
+
+Cylc commands identify workflows via their names, which are relative path names
+under the :term:`cylc run directory`, ``~/cylc-run/`` by default.
+
+Workflows can be grouped together under sub-directories. E.g.:
+
+.. code-block:: console
+
+   $ cylc scan --state=all --name nwp
+   nwp
+    |-oper
+    | |-region1  Local Model Region1       /home/oliverh/cylc-run/nwp/oper/region1
+    | `-region2  Local Model Region2       /home/oliverh/cylc-run/nwp/oper/region2
+    `-test
+      `-region1  Local Model TEST Region1  /home/oliverh/cylc-run/nwp/test/region1
```

Comment:
I don't think `cylc scan` can produce output like that, the closest it gets is with `--format=tree`.